### PR TITLE
Improvement Fab refactoring rc0

### PR DESCRIPTION
**Pull request by ramonornela**
#### Short description of what this resolves:
Improvements of code and docs after refactoring rc0

#### Changes proposed in this pull request:
- Delete @input fab;
- Adjust docs usage fab to ion-fab;
- Delete variables SASS not used;
- Delete @property fab-* docs;
- Link docs FabButton and FabContainer;
- Delete tests fab.

**Ionic Version**: rc0

### CODE REVIEW

File: src/components/button/test/raised/main.html [hunk 1]
```diff
@@ -25,8 +25,8 @@
   </p>
 
   <p>
-    <button ion-button raised fab style="position: relative">FAB</button>
-    <button ion-button raised fab style="position: relative" class="activated">FAB</button>
+    <button ion-button raised ion-fab style="position: relative">FAB</button>
```
Comment (manucorporat) [inline src/components/button/test/raised/main.html]: First of all, thanks for the PR. Everything makes a lot of sense! just one change: I think you could remove this paragraph completely ^.

File: src/components/button/button.ts [hunk 2]
```diff
@@ -61,7 +54,7 @@ import { isTrueProperty } from '../../util/util';
   *
   *  <button ion-button round outline>Outline + Round</button>
   *
-  *  <button ion-button fab outline>FAB</button>
+  *  <button ion-button ion-fab outline>FAB</button>
```
Comment (manucorporat) [inline src/components/button/button.ts]: Same here, remove this line ^

File: src/components/button/button.ts [hunk 3]
```diff
@@ -52,7 +45,7 @@ import { isTrueProperty } from '../../util/util';
   *
   *  <button ion-button round>Round Button</button>
   *
-  *  <button ion-button fab>FAB</button>
+  *  <button ion-button ion-fab>FAB</button>
```
Comment (manucorporat) [inline src/components/button/button.ts]: And here.

File: src/components/button/button.ts (hunk 2 above)
Comment (ramonornela) [inline src/components/button/button.ts]: Removed all, thanks by report @manucorporat 😉

**Comment by manucorporat** [comment]
LGTM! I will squash the changes into one commit!

**Comment by manucorporat** [comment]
@ramonornela merged! thanks

**Pull request merged**